UM/View/GL/OpenGL.py

```diff
@@ -4,8 +4,11 @@
 import sys
 import ctypes   # type: ignore
 
-from PyQt5.QtGui import QOpenGLVersionProfile, QOpenGLContext, QOpenGLFramebufferObject, QOpenGLBuffer
-from PyQt5.QtWidgets import QMessageBox
+from PyQt6.QtOpenGL import QOpenGLVersionProfile, QOpenGLVersionFunctionsFactory
```

Comment:
```suggestion
from PyQt6.QtOpenGL import QOpenGLVersionFunctionsFactory
```
Remove unused import